samples/snippets/conftest.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import datetime
+import uuid
+
+import google.auth
+from google.cloud import bigquery
+from google.cloud import datacatalog_v1
+
+import pytest
+
+bigquery_client = bigquery.Client()
```

Comment:
Should this be a (session-scoped) fixture? It could use the existing `credentials` and `project_id` fixtures to prevent multiple auth flows.